examples/asr/speech_to_text.py

```diff
@@ -65,6 +89,7 @@ def main(args):
     parser = pl.Trainer.add_argparse_args(parser)
     parser = add_asr_args(parser)
     parser = add_optimizer_args(parser)
+    parser = add_scheduler_args(parser)
```

Comment:
why are you adding it to plTrainer parser?

Reply:
This is to add the lr scheduling parameters to argparse. Almost all of our schedulers depend on 4-5 common parameters so this will include them. Special ones will still need to add the args though.